project/perform-release.sh

```diff
@@ -9,10 +9,15 @@ if [[ $1 != "patch" && $1 != "minor" && $1 != "major" ]] ; then
   exit 1;
 fi
 
-echo ""
-echo "Sign into npm"
-echo ""
-npm whoami &>/dev/null || npm login
+currentBranch="$(git symbolic-ref HEAD 2>/dev/null)" ||
+currentBranch="(unnamed currentBranch)"     # detached HEAD
+
+currentBranch=${currentBranch##refs/heads/}
```

Comment:
No, no - I just wanted to be able to use this feature myself :)

I did just find a doc at http://tldp.org/LDP/abs/html/string-manipulation.html which can be added as a comment, though.